### PR TITLE
Un-hide the Pontus-X Testnet layer

### PR DESCRIPTION
**Pull request by csillag**
This PR enabled showing the Pontus-X Testnet in the layer selector.

Please note that it's still not on the graph.

(I'm not sure how much of a problem that is.)

### CODE REVIEW

File: src/app/utils/route-utils.ts [hunk 1]
```diff
@@ -37,6 +37,14 @@ export type SpecifiedPerEnabledLayer<T = any, ExcludeLayers = never> = {
 
 export type SpecifiedPerEnabledRuntime<T = any> = SpecifiedPerEnabledLayer<T, typeof Layer.consensus>
 
+export const hiddenLayers: Layer[] = [Layer.pontusxdev]
+
+console.log('Hidden layers are', hiddenLayers)
```
Comment (buberdds) [inline src/app/utils/route-utils.ts:42]: remove

File: src/app/utils/route-utils.ts [hunk 2]
```diff
@@ -37,6 +37,14 @@ export type SpecifiedPerEnabledLayer<T = any, ExcludeLayers = never> = {
 
 export type SpecifiedPerEnabledRuntime<T = any> = SpecifiedPerEnabledLayer<T, typeof Layer.consensus>
 
+export const hiddenLayers: Layer[] = [Layer.pontusxdev]
```
Comment (buberdds) [inline src/app/utils/route-utils.ts:40]: what's a plan for pontusxdev? It's going to be accessible via direct url only?

File: src/app/utils/route-utils.ts (hunk 1 above)
Comment (csillag) [reply src/app/utils/route-utils.ts]: Done.

File: src/app/utils/route-utils.ts (hunk 2 above)
Comment (csillag) [reply src/app/utils/route-utils.ts:40]: Yes, in our release, it's only available via direct URL. (On the pontus-x branch, it's not hidden, so it's available in the layer selector.)

**Review by buberdds — APPROVED** [review]

**Pull request merged**